packages/tools/src/tools/segmentation/RectangleROIStartEndThresholdTool.ts

```diff
@@ -264,6 +267,78 @@ class RectangleROIStartEndThresholdTool extends RectangleROITool {
     data.cachedStats.projectionPointsImageIds = projectionPointsImageIds;
   }
 
+  _computePointsInsideVolume(annotation, imageVolume, enabledElement) {
```

Comment:
can you please add a small doc on top of this function on what it actually does?